.travis.yml

```diff
@@ -1,21 +1,27 @@
-# Config file for automatic testing at travis-ci.org
-
 language: python
-
+sudo: false
 python:
   - "3.5"
   - "3.4"
   - "2.7"
 
-before_install:
-  - pip install codecov
+before_script:
+  - flake8 verification/ --max-line-length=120
+
+cache:
+  directories:
+  - $HOME/.pip-cache/
 
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -r requirements-test.txt
+install:
+  - pip install -r requirements-test.txt --allow-all-external --download-cache $HOME/.pip-cache
 
-# command to run tests using coverage, e.g. python setup.py test
-script: coverage run --source verification runtests.py
```

Comment:
I rather want to use coverage in this one, so please revert this :) (don't forget to add coverage to requirements-test)